src/expression/arithmetic/__test__/unit/multiply.test.ts

```diff
@@ -0,0 +1,106 @@
+import { notSimplified, operand } from '../../../../__test__/helpers'
+import { Result } from '../../../../common/evaluable'
+import { Operand } from '../../../../operand'
+import { Value } from '../../../../operand/value'
+import { defaultOptions } from '../../../../parser/options'
+import { Multiply, OPERATOR } from '../../multiply'
+
+describe('Expression - Arithmetic - Multiply', () => {
+  describe('constructor', () => {
+    test.each([[[]], [[operand(5)]]])('arguments %p should throw', (args) => {
+      expect(() => new Multiply(...args)).toThrowError(
+        'multiply expression requires at least 2 operands'
+      )
+    })
+  })
+
+  const testCases: [Result, ...Operand[]][] = [
+    [200, operand(20), operand(10)],
+    [6000, operand(10), operand(20), operand(30)],
+    [-200, operand(10), operand(-20)],
+    [-200, operand(-10), operand(20)],
+    [200, operand(-10), operand(-20)],
+    [0, operand(10), operand(0)],
+    [-0, operand(0), operand(-10)],
+    [0, operand(0), operand(0)],
+    [3.36, operand(2.8), operand(1.2)],
+    [2.592591962963, operand(2.333333), operand(1.111111)],
+    [0.04000000000000001, operand(0.4), operand(0.1)],
```

Comment:
is this really expected?

Reply:
the max decimal places of a multiply result is the number of left's decimal places + the number of right's
```
> 0.13*0.4
0.052000000000000005
> 0.123*0.234
0.028782000000000002
```
the first result should be 0.052
the 2nd should be 0.028782

Reply:
I can apply the same concept here, but thought it was overkill and unnecessary. See [this](https://github.com/briza-insurance/illogical/pull/243#discussion_r1702175758).

I don't think it's likely that a division and multipliaction will be followed by an Equality. Although that's really specific to our context.

I can work on it